### ISSUE TITLE
Youngstown, Ohio

### ISSUE THREAD

**Issue by deathandminimalism** [defund12/defund12.org#560]
To: mayor@youngstownohio.gov, vmarrow@youngstownohio.gov, dkitchen@youngstownohio.gov, joliver@youngstownohio.gov, jhughes@youngstownohio.gov, sturner@youngstownohio.gov, mikeray@youngstownohio.gov,  lmcnally@youngstownohio.gov, adavis@youngstownohio.gov, badamczak@youngstownohio.gov, chieflees@youngstownohio.gov

Subject: Defund the Youngstown Police Department

My name is [YOUR NAME] and I am a resident of Youngstown, Ohio. 

The City of Youngstown must do better by its residents. I am urging you to divest from the criminalization of our communities and reduce police spending in the budget for the 2021 fiscal year. The City of Youngstown has already announced about deep cuts in the very departments that our community needs most (parks and rec, public works, community development), while the Youngstown Police Department budget has continued to climb.  

In 2020, the Youngstown Police Department budget increased to almost $18 million dollars, considerably larger than any other city department. With this excessive spending, our streets are not safer. 

Now especially, the Mayor and City Council can take a stand for racial justice by significantly defunding policing and investing in Black and Brown communities–starting with cutting the least transparent and most harmful parts of the Youngstown Police Department budget.

Racist police violence doesn’t just happen in other cities–it happens here in Youngstown. We see it in the disproportionate imprisonment of black people, and in the racialized inequality that is prevalent in the Mahoning Valley. The City Council must stop investing in targeted criminalization and surveillance, and fund what Black and Brown communities need to be safe and healthy: COVID-19 relief, housing, healthcare, treatment, healing, access to fresh and healthy food, community centers, among other things.

Thank you,
[YOUR NAME]
[YOUR ADDRESS]
[YOUR EMAIL]
[YOUR PHONE NUMBER]

**Comment by ctneal91** [comment]
on it now

**Comment by teddywilson** [comment]
@ctneal91 per TEAMS.md, we are following that the content team will take a look before making any PR.

**Comment by ctneal91** [comment]
Yep. @teddywilson I see that now. @milesalex pointed that out as well. Noted from here on out.

**Comment by skullface** [comment]
Hi @deathandminimalism, thanks for submitting Toledo to the site. I’m a content reviewer (and fellow Ohioan!) and I have suggested a few changes to [align with the defund12.org style guide](https://github.com/defund12/defund12.org/blob/gh-pages/EMAIL_TEMPLATE_STYLE_GUIDE.md) and general readability:

---

The City of Youngstown must do better by its residents. I urge you to divest from the criminalization of Black communities and reduce police spending in the budget for the 2021 fiscal year. 

The City of Youngstown has already announced about deep cuts in the very departments that our community needs most, like parks and recreation, public works, and community development. However, the Youngstown Police Department budget has continued to climb. In 2020, the Youngstown Police Department budget increased to almost $18 million dollars, a considerably larger amount than any other department. With this excessive spending, our streets are not safer.

Now more than ever, the Mayor and City Council should take a stand for racial justice. Defund policing and invest in Black and Brown communities, starting with cutting the least transparent and most harmful parts of the Youngstown Police Department budget.

Racist police violence doesn’t just happen in other cities—it happens here in Youngstown. We see it in the disproportionate imprisonment of Black people and in the racialized inequality prevalent in the Mahoning Valley. The City Council must stop investing in targeted criminalization and surveillance, and instead fund what Black and Brown communities need to be safe and healthy: programs like COVID-19 relief, housing, healthcare, treatment, healing, access to fresh and healthy food, community centers.

---

Additionally, we need some clarification.

> In 2020, the Youngstown Police Department budget increased to almost $18 million dollars, considerably larger than any other city department. With this excessive spending, our streets are not safer.

* Earlier in the message "fiscal year 2021" is mentioned, but this sentence uses "2020". Whichever phrase is more accurate should be used in both places.
* Can you add specificity here? Is there a discrepancy you find particularly egregious you can compare, to better draw attention to the disparity? Is the **increase** larger, or the amount larger, or both?

> The City Council must stop investing in targeted criminalization **and surveillance**, and fund what Black and Brown communities need to be safe and healthy: COVID-19 relief, housing, healthcare, treatment, healing, access to fresh and healthy food, community centers, among other things.

Is surveillance a particular issue in Youngstown? Has there been recent legislation or technology introduced regarding law enforcement surveillance? Just want to make sure we’re being specific to the locale!

---

Thanks again!

**Comment by velvetoveralls** [comment]
@skullface hello! came here to start the youngstown template and excited to see it already open! 

I imagine the 2020 vs 2021 was framed that was because the 2020 budget is already planned, so we're asking for them to make these changes to the 2021 budget. 

I'm not sure where to find the budgets, or I'd be happy to add a comparison to education, housing, etc myself. @deathandminimalism where did you find these?

thanks to everyone working on this!!

**Comment by deathandminimalism** [comment]
> @skullface hello! came here to start the youngstown template and excited to see it already open!
> 
> I imagine the 2020 vs 2021 was framed that was because the 2020 budget is already planned, so we're asking for them to make these changes to the 2021 budget.
> 
> I'm not sure where to find the budgets, or I'd be happy to add a comparison to education, housing, etc myself. @deathandminimalism where did you find these?
> 
> thanks to everyone working on this!!

The 2020 budget can be found [here](https://www.wkbn.com/wp-content/uploads/sites/48/2020/03/2020-Appropriation-Budget-2.pdf).

**Comment by deathandminimalism** [comment]
@skullface I have updated the original post with your new language. The 2021 budget for the City has not been determined (they might get it passed by March 2021, if they're lucky). While fiscal 2020 has already been allocated, fiscal 2021 is where Council has the most leverage. 

While traditional surveillance is not an issue in Youngstown (cameras are rare at best), our majority-minority neighborhoods are considerably over-policed (specifically the South side).

I have removed/updated that specific line in the draft language.  

Thank you
